src/main/java/mate/controller/car/GetAllCarsController.java

```diff
@@ -3,13 +3,15 @@
 import java.io.IOException;
 import java.util.List;
 import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import mate.lib.Injector;
 import mate.model.Car;
 import mate.service.CarService;
 
+@WebServlet(urlPatterns = "/cars/")
```

Comment:
```suggestion
@WebServlet(urlPatterns = "/cars")
```

lets make all urls uniformed

Reply:
done